test/index.test.ts

```diff
@@ -261,4 +261,29 @@ describe('Successful tests', () => {
         const isReportFileExist = fs.existsSync(getPathToCreatedReport('shouldNotBeSaved.html'));
         expect(isReportFileExist).toBe(false);
     });
+
+    it('Raw AxeResults passed and generates report with custom template', async () => {
+        const customSummary = `Test Case: Full page analysis
+        <br>Steps:</br>
+        <ol style="margin: 0">
+        <li>Open https://dequeuniversity.com/demo/mars/</li>
+        <li>Analyze full page with all rules enabled</li>
+        <li>Custom template is used</li>
+        </ol>`;
+
+        const reportFileName = 'custom_index.html';
+        const templatePath = path.join(__dirname, 'pageTemplateOnlyBody.html');
+        const outputDir = 'docs';
```

Comment:
It would be better to save the results of test files into the `artifacts` folder instead of `docs` because `docs` is meant only for `index.html` that is generated by tests but used as source of https://lpelypenko.github.io/axe-html-reporter/. HTML report is saved `artifacts` by default.

It is easy to do just by removing `const outputDir = 'docs';` line and not passing it in `createHtmlReport` like this:

```
const templatePath = path.join(__dirname, 'pageTemplateOnlyBody.html');
createHtmlReport({
    results: rawAxeResults,
    options: { projectKey: 'DEQUE', 
    customSummary, reportFileName, templatePath },
});
expect(
    fs.readFileSync(getPathToCreatedReport(reportFileName), {
        encoding: 'utf8',
    })
).toMatchSnapshot();
```

Thank you!